postgres/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 14.2.4 / 2023-09-07
+
+***Fixed***:
+
+* Initialize pg_settings on Postgres check start and lazy load pg_settings if it's not set ([#15773](https://github.com/DataDog/integrations-core/pull/15773))
```

Comment:
I know we've covered this recently (tagging @FlorentClarret and @yzhan289), but I'm confused again: This change shows up in the previous release on the master branch. Should we be adding it as a separate release again?

Reply:
Interesting, I see [this in the master changelog](https://github.com/DataDog/integrations-core/blob/c758504930a54f90c4d35d2d4b5e6868415b6106/postgres/CHANGELOG.md?plain=1#L9), looks identical to the line we're adding in this PR.

Reply:
Yeah, I see it now. Looks like the developer added it to unreleased as part of their [pr](https://github.com/DataDog/integrations-core/pull/15773/files#diff-b22dfd73c4dd870b01b51f3c8a21964e1ea48ebc96a8b7214dd1e396c6344e71R9) but in master it came under [14.2.3 release](https://github.com/DataDog/integrations-core/blob/c758504930a54f90c4d35d2d4b5e6868415b6106/postgres/CHANGELOG.md?plain=1#L9)

I have removed this fix from 14.2.3 as part of this pr. Thanks for noticing the duplicate.

But we need to investigate the reason for this.

Reply:
I think right before merging they got conflicts and resolved them in such a way that it landed in a release that happened. Here's their final commit into master branch:
https://github.com/DataDog/integrations-core/commit/377db0d98526ae1cf09a17d5438439bfc4c7becf

Reply:
It is in the same [link](https://github.com/DataDog/integrations-core/pull/15773/files#diff-b22dfd73c4dd870b01b51f3c8a21964e1ea48ebc96a8b7214dd1e396c6344e71R9) that I referenced in [earlier comment](https://github.com/DataDog/integrations-core/pull/15778#discussion_r1318250386). I didn't expand the earlier lines to confirm rather assumed it is in unreleased section. Oversight from my side.